proposals/lip-define_new_transaction_schema.md

```diff
@@ -98,6 +100,7 @@ transactionSchema = {
         },
         "senderPublicKey": {
             "dataType": "bytes",
+            "length": PUBLIC_KEY_LENGTH_BYTES,
```

Comment:
For this constant I would use `ED25519_PUBLIC_KEY_LENGTH` to be in line with other LIPs.

Reply:
Fixed